src/main/java/codechicken/nei/util/NBTJson.java

```diff
@@ -216,8 +216,8 @@ protected static NBTBase restoreEmptyList(JsonObject obj) {
         if (obj.has("__custom_type")) {
             try {
                 final String className = obj.get("__custom_type").getAsString();
-                return (NBTBase) Class.forName(className).newInstance();
-            } catch (ClassNotFoundException | InstantiationException | IllegalAccessException th) {}
+                return (NBTBase) Class.forName(className).getConstructor().newInstance();
+            } catch (Exception th) {}
```

Comment:
I'd prefer the explicit exceptions being called out vs a generic `Exception` catch...